cloudoperators/juno
# feat(mock-server): add mock-server package and detach juno-utils usage.

# Summary

This pull request move out dependency from  the `juno-utils` package,  to a new package `@cloudoperators/juno-mock-server`, and updates its usage within our applications (`greenhouse`, `supernova`, `example`, and `doop`).

# Changes Made

-  Create a new package called  `@cloudoperators/juno-mock-server` which use latest greatest technology and tranfer whole logic to TS
- Updated all instances of the hook in the following applications:
  - `greenhouse`
  - `supernova`
  - `example`
  - `doop`

Maybe we need to reconsider the whole implementation and just use simple usage of json-server + faker.js but still keep it in our package... I would also reconsider it as a whole process inside the apps how we use this mock server instead of whole logic I would simply change url to a mock server and start it in a dev mode...

@ArtieReus lets discuss it. 

# Related Issues

- #446 

# Testing Instructions

1. Run `npm i` to ensure dependencies are up-to-date.
2. Start the application with `turbo dev --filter=supernova` and try to start it with mock property set to true.
3. Any action should go to a mock logic server


# Checklist
